Dockerfile

```diff
@@ -1,12 +1,16 @@
-FROM ruby:2.3.5-jessie
-RUN apt-get update -qq && apt-get install -y build-essential libpq-dev
-RUN curl -sS http://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
-RUN echo "deb http://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
-RUN curl -sL https://deb.nodesource.com/setup_8.x | bash -
-RUN apt-get update -qq && apt-get install -y yarn nodejs
-RUN mkdir -p /app
+FROM ruby:2.6.5-jessie
```

Comment:
The removal of layers look good, but I think we're actually using the lower version of Ruby in production, so it would be good to keep those aligned. In fact, I think the app doesn't work at all with the newer version. (I see you haven't checked the box that says you tested it locally - if not, and it _does_ work, we'd probably want to update the version in production as well.)

Reply:
I plan to test locally, just trying to get docker to start.

Reply:
Great, let me know when we can check again.

Reply:
You can check again now!